fiftyone/utils/torch.py

```
@@ -2605,6 +2667,245 @@ def _parse_inputs(
+        if worker_info is None:
+            video_indices = range(len(self.video_paths))
+        else:
+            worker_id = worker_info.id
+            num_workers = worker_info.num_workers
+            video_indices = range(
+                worker_id, len(self.video_paths), num_workers
+            )
+
+            logger.info(
+                f"Worker {worker_id}/{num_workers} assigned "
+                f"{len(list(video_indices))} videos"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)